src/test/java/org/kohsuke/github/EnterpriseManagedSupportTest.java

```diff
@@ -0,0 +1,184 @@
+package org.kohsuke.github;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.hamcrest.Matchers.*;
+
+// TODO: Auto-generated Javadoc
+
+/**
+ * The Class EnterpriseManagedSupportTest.
+ */
+public class EnterpriseManagedSupportTest extends AbstractGitHubWireMockTest {
+
+    private static final String NOT_PART_OF_EXTERNALLY_MANAGED_ENTERPRISE_ERROR = "{\"message\":\"This organization is not part of externally managed enterprise.\","
+            + "\"documentation_url\": \"https://docs.github.com/rest/teams/external-groups#list-external-groups-in-an-organization\"}";
+
+    private static final String UNKNOWN_ERROR = "{\"message\":\"Unknown error\","
+            + "\"documentation_url\": \"https://docs.github.com/rest/unknown#unknown\"}";
+
+    private static final String TEAM_CANNOT_BE_EXTERNALLY_MANAGED_ERROR = "{\"message\":\"This team cannot be externally managed since it has explicit members.\","
+            + "\"documentation_url\": \"https://docs.github.com/rest/teams/external-groups#list-a-connection-between-an-external-group-and-a-team\"}";
+
+    /**
+     * Test to ensure that only HttpExceptions are handled
+     *
+     * @throws IOException
+     *             Signals that an I/O exception has occurred.
+     */
+    @Test
+    public void testIgnoreNonHttpException() throws IOException {
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+
+        final GHException inputCause = new GHException("Cause");
+        final GHException inputException = new GHException("Test", inputCause);
+
+        final Optional<GHException> maybeException = EnterpriseManagedSupport.forOrganization(org)
+                .handleException(inputException);
```

Comment:
I think `handleException` is more like `filterException`.  Consider rename, but I'm open to leaving if you feel strongly about it.

Reply:
Find for me. Renamed.